src/Windows/dll/dcap_provider.vcxproj

```diff
@@ -111,9 +108,9 @@
   <Target Name="GetPrerequisitesFromIntel" BeforeTargets="PrepareForBuild">
     <Exec Command="powershell ..\get-prereqs.ps1" />
   </Target>
-  <!--Target Name="RunUnitTest" AfterTargets="Build" DependsOnTargets="GetPrerequisitesFromIntel;EnsureNuGetPackageBuildImports">
-    <Exec ContinueOnError="True" Command="xcopy /y ..\UnitTest\$(Platform)\$(Configuration)\dcap_provider_ut.exe $(OutDir)"/>
-    <Exec Command="$(OutDir)\dcap_provider_ut.exe" />
+  <!--Target Name="RunUnitTest" AfterTargets="Build" DependsOnTargets="GetPrerequisitesFromIntel;EnsureNuGetPackageBuildImports">
+    <Exec ContinueOnError="True" Command="xcopy /y ..\UnitTest\$(Platform)\$(Configuration)\dcap_provider_ut.exe $(OutDir)"/>
+    <Exec Command="$(OutDir)\dcap_provider_ut.exe" />

```

Comment:
Can you please remove commented out code?

Reply:
These were commented out to begin with.  I am not sure why it needs to be uncommented in this PR?